lib/core/transpiler_session/transpiler_session.h

```diff
@@ -45,22 +45,29 @@ struct TranspilerSession {
 
     const UserInput& getInput() const { return _input; }
 
-    UserInput& getInput() { return _input; }
-
     const UserOutput& getOutput() const { return _output; }
 
     UserOutput& getOutput() { return _output; }
 
-    void moveOutputToInput() {
-        _input.source = _output.normalized.source;
-        _input.headers = _output.normalized.headers;
+    const std::string& getSource() const { return _source; }
+    std::string& getSource() { return _source; }
+
+    const std::map<std::string, std::string>& getHeaders() const { return _headers; }
```

Comment:
Add 'Staged' name before -> 'getStagedHeaders' or something else to explain pipeline nature of those files

Reply:
Done